vm-migration/src/lib.rs

```diff
@@ -279,4 +289,16 @@ pub trait Transportable: Pausable + Snapshottable {
 /// and Snapshottable.
 /// Moreover a migratable component can be transported to a remote or local
 /// destination and thus must be Transportable.
-pub trait Migratable: Send + Pausable + Snapshottable + Transportable {}
+pub trait Migratable: Send + Pausable + Snapshottable + Transportable {
+    fn start_migration(&mut self) -> std::result::Result<(), MigratableError> {
+        Ok(())
+    }
+
+    fn stop_migration(&mut self) -> std::result::Result<(), MigratableError> {
+        Ok(())
+    }
+
+    fn dirty_ranges(&mut self) -> std::result::Result<MemoryRangeTable, MigratableError> {
```

Comment:
Nit: would be better to rename as `get_dirty_log()` to stay consistent with what we have from the `hypervisor` crate?

Reply:
I can change it to `dirty_log()`, but `get_dirty_log()` is actually wrong in the hypervisor crate. Adding `get_` is redundant to the function name, which is self explanatory about the fact that we're trying to retrieve the dirty logs.